fs-integration-test-suite/accelerator-test-framework/pom.xml

```diff
@@ -125,6 +125,12 @@
             <groupId>com.fasterxml.uuid</groupId>
             <artifactId>java-uuid-generator</artifactId>
         </dependency>
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+            <version>1.2</version>
```

Comment:
lets maintain the versions separately like we have done in other places

Reply:
Fixed